storey/table.py

```diff
@@ -79,12 +83,14 @@ async def close(self):
         await self._storage.close()
 
     async def _aggregate(self, key, data, timestamp):
+        key = get_hashed_key(key)
         if not self._schema:
             await self._get_or_save_schema()
 
         self._get_aggregations_attrs(key).aggregate(data, timestamp)
 
     async def _get_features(self, key, timestamp):
+        key = get_hashed_key(key)
```

Comment:
it's not needed here, it calls _get_aggregations_attrs that already does the hashing